lab-michael/test/test-greeting.js

```diff
@@ -0,0 +1,9 @@
+var greet = require('../greeting.js');
+var assert = require('assert');
+
```

Comment:
Mocha tests are typically structured like this:
- `describe` the module
  - `describe` the method
    - `it` does something: test it
    - `it` does something else: test it
etc, etc...